src/shims/sig.rs

```diff
@@ -53,6 +53,8 @@ macro_rules! shim_sig_arg {
             "*const _" => $this.machine.layouts.const_raw_ptr.ty,
             "*mut _" => $this.machine.layouts.mut_raw_ptr.ty,
             ty if let Some(libc_ty) = ty.strip_prefix("libc::") => $this.libc_ty_layout(libc_ty).ty,
+            ty if let Some(win_ty) = ty.strip_prefix("winapi::") =>
```

Comment:
This is shorthand for a longer path used by the windows_ty_layout function, I thought this made usage a lot nicer and more readable.